api/bases/watcher.openstack.org_watchers.yaml

```diff
@@ -97,6 +181,7 @@ spec:
                 type: string
             required:
             - apiContainerImageURL
+            - apiServiceTemplate
```

Comment:
Do we want to add a webhook validation for `apiServiceTemplate`? Or Can I include it in this this pr https://github.com/openstack-k8s-operators/watcher-operator/pull/33 ?

Reply:
I'd bet by adding more validation later, once we have a final definition. I'd propose to keep a basic skeleton of the validation in https://github.com/openstack-k8s-operators/watcher-operator/pull/33 and complete it when we have the operator in a functional status.